README.md

```diff
@@ -215,6 +217,98 @@ the following in [.travis.yml](.travis.yml).
+1. Name your new item something like **ProfessorM**.  
+1. Select **Freestyle project** and click **OK**. The item configuration page will display.
+1. In the **Source Code Management** section, select **Git.**
+1. Input your Repository URL (for example, `https://github.com/sugarcrm/school`) and add your credentials.
+1. Configure the Build Triggers section as you'd like. If you do not want to configure this now, you can skip this step
+and manually trigger the builds instead.
+1. You do not need to configure anything in the Build Environment section.
+1. In the **Build** section, click **Add build step** and select **Execute shell**. 
+1. In the **Command** box that appears, input the following: `bash -ex buildPackageInJenkins.sh`
+1. In the **Post-build Actions** section, click **Add post-build action** and select **Archive the artifacts**.
+1. In the **Files to archive** box that appears, input the following: `package/releases/*.zip`
+1. In the **Post-build Actions** section, click **Add post-build action** and select **Delete workspace when build is done**.
+1. Click **Save**.
+
+### Running a build and viewing the results
+
+Once you have your Jenkins project created, it's time to see if it works!
+
+Navigate to your ProfessorM project in Jenkins. Trigger your build manually by clicking **Build Now**.
+
+View the Build History panel to quickly browse the build results.  If you see a blue dot, you know all of the tests passed!
+
+![Build passed](images/jenkins-buildpassed.png)
+
+Click on a build to view more details.  If the build passed, the Professor M module loadable packages will be stored as 
+build artifacts.
+
+![Build artifacts](images/jenkins-buildartifact.png)
+
+To see more details on a build, click **Console Output**.  If any step in the process fails (for example, a Jasmine test
+fails), the remaining steps will not be run.  If everything succeeds, you'll be able to find the Jasmine test results,
+the PHPUnit test results, and the results of the Professor M module loadable packages being generated, copied, and archived. 
+
+Jasmine results:
+
+![Jasmine passed](images/jenkins-jasmine.png)
+
+PHPUnit results:
+
+![PHPUnit results](images/jenkins-phpunit.png)
+
+Professor M package results:
+
+![ProfM zip](images/jenkins-profm.png)
+
+Tip:  If you see errors in the Console Output about git not being found, you may need to update your Jenkins configuration.
+On the Jenkins dashboard, click **Manage Jenkins**. Click **Conifgure System**. In the **Global Properties** section,
+enable the **Tool Locations** checkbox. In the Name box, select **(Git) Default**.  In the Home box, input the path
+to the Git installation (for example, `/usr/bin/git`).  Click **Save**.
+
+Tip:  If you see errors in the Console Output similar to 'Fatal error: Unable to find local grunt', you may need to 
+update your copy of [buildPackageInJenkins.sh](buildPackageInJenkins.sh).  The likely cause is that your 
```

Comment:
You might consider stating here that this issue likely would only affect users running Jenkins in Docker

Reply:
Good point! I've updated.